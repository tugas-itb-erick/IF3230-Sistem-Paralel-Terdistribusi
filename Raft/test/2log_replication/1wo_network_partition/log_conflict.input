5
set_timeout 1 0
simulate 10
client_send_request 1 + 5
client_send_request 1 * 5
client_send_request 1 - 2
crash 1
set_timeout 2 0
simulate 10
client_send_request 2 - 5
client_send_request 2 + 4
client_send_request 2 * 3
simulate 100
get_leader_state 2
get_state 2
get_state 3
get_state 4
get_state 5
restart 1
simulate 100
get_leader_state 2
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
end_simulation
