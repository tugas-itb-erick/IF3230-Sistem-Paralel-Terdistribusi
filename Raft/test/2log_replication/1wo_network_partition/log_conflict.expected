L2 next_index[1 X 4 4 4 ] match_index[0 X 3 3 3 ]
S2 leader term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S3 follower term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S4 follower term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S5 follower term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
L2 next_index[4 X 4 4 4 ] match_index[3 X 3 3 3 ]
S1 follower term:3 voted_for:-1 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S2 leader term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S3 follower term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S4 follower term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
S5 follower term:3 voted_for:2 commit_index:3 data:-3 logs:T0[0] T3[-5+4*3]
